project/Build.scala

```diff
@@ -327,7 +325,7 @@ object DottyBuild extends Build {
         partestLockFile.createNewFile
         partestLockFile.deleteOnExit
       },
-      runPartestRunner <<= Def.inputTaskDyn {
+      runPartestRunner := Def.inputTaskDyn {
```

Comment:
If you change this, you also need to add `.value` after the closing `}` corresponding to the `{` on this line. This could explain the problem that you observe.